rms/search_engine/views.py

```diff
@@ -1,10 +1,33 @@
 from fastapi import APIRouter
 from rms.search_engine.models import SearchBody, SearchResponse
-from rms.search_engine.services import search_scopus_service
+from rms.search_engine.models.dblp_models import DblpAuthorSearchBody, DblpAuthorResponse
+from rms.search_engine.models.scholar_models import ScholarAuthorSearchBody, ScholarAuthorResponse
+from rms.search_engine.services import search_scopus_service, get_author_dblp_service, get_author_scholar_service
+from rms.utils.exceptions import NotFound
 
 router = APIRouter()
 
 
 @router.post("/discover_reviewers")
 async def search_scopus(body: SearchBody) -> SearchResponse:
     return await search_scopus_service(body)
+
+
+@router.get("/get_author_dblp")
+async def get_author_dblp(body: DblpAuthorSearchBody) -> DblpAuthorResponse:
+    dblp_author = await get_author_dblp_service(body)
+
+    if not dblp_author:
```

Comment:
i would return response with code 200 and empty string - not code 404

Reply:
fixed